packages/gdl-frontend/lib/initSentry.js

```diff
@@ -18,6 +18,7 @@ const {
 export default function() {
   if (process.env.NODE_ENV === 'production' && REPORT_ERRORS) {
     Sentry.init({
+      release: 'test-release',
```

Comment:
Burde vi da ikke sette verdien av denne til å faktisk være git-commit hashen? Hvis dette skal gi noe mening?

Reply:
Tanken var å bruke githashen når vi har fått til flyten for å gjøre dette automatisk, men jeg skal ta å sette den manuelt her jeg!

Reply:
Hmm...om du ser i `build.sh` for gdl-frontend, så er $VERSION der git-hashen. Den brukes når man lager docker-imaget, og jeg regner med at den kan brukes her også på en eller annen måte? Test det gjerne ut sånn dere har gjort det nå, men kanskje dere slipper å finne opp hjulet når dere skal få inn git-hashen senere.